explorer/client/src/utils/internetapi/mock_transactions.json

```diff
@@ -0,0 +1,47 @@
+[
```

Comment:
Given this is static JSON data, the directory `explorer/client/src/utils/static` would be a more suitable directory. The `findDataFromID` function in `src/utils/static/searchUtil` can then be updated to search across the multiple different static JSON files.